rad/src/operators/float.rs

```diff
@@ -42,3 +59,183 @@ pub fn less_than(input: &RadonFloat, args: &[Value]) -> Result<RadonBoolean, Rad
     let other = from_value::<f64>(arg).map_err(|_| wrong_args())?;
     Ok(RadonBoolean::from(input.value() < other))
 }
+
+pub fn modulo(input: &RadonFloat, args: &[Value]) -> Result<RadonFloat, RadError> {
+    let wrong_args = || RadError::WrongArguments {
+        input_type: RadonFloat::radon_type_name(),
+        operator: "Modulo".to_string(),
+        args: args.to_vec(),
+    };
+
+    let arg = args.first().ok_or_else(wrong_args)?.to_owned();
+    let modulo = from_value::<f64>(arg).map_err(|_| wrong_args())?;
+    Ok(RadonFloat::from(input.value() % modulo))
+}
+
+pub fn negate(input: &RadonFloat) -> RadonFloat {
+    RadonFloat::from(-input.value())
+}
+
+pub fn power(input: &RadonFloat, args: &[Value]) -> Result<RadonFloat, RadError> {
+    let wrong_args = || RadError::WrongArguments {
+        input_type: RadonFloat::radon_type_name(),
+        operator: "Power".to_string(),
+        args: args.to_vec(),
+    };
+
+    let arg = args.first().ok_or_else(wrong_args)?.to_owned();
+    let exp = from_value::<i32>(arg).map_err(|_| wrong_args())?;
+
+    Ok(RadonFloat::from(input.value().powi(exp)))
```

Comment:
Why is the argument an integer? You could use floating point exponentiation with `powf` here.